lib/libzpool/kernel.c

```diff
@@ -783,7 +783,8 @@ dprintf_setup(int *argc, char **argv)
  * =========================================================================
  */
 void
-__dprintf(const char *file, const char *func, int line, const char *fmt, ...)
+__dprintf(boolean_t dprint, const char *file, const char *func,
```

Comment:
I did consider it, but in the kernel these functions are really almost exactly the same. Since the kernel implementation is the primary one, I wanted this code to match that.